finagle-redis/src/main/scala/com/twitter/finagle/redis/protocol/commands/Lists.scala

```diff
@@ -62,6 +62,10 @@ case class LTrim(key: Buf, start: Long, end: Long) extends ListRangeCommand {
   def name: Buf = Command.LTRIM
 }
 
+case class RPopLPush(source: Buf, dest: Buf) extends MoveCommand {
```

Comment:
i /think/ `dest` should be named `destination` in order to satisfy the `MoveCommand` trait.

Reply:
 done